tests/bayesian/test_weight_explosion_averages.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import numpy as np
+
+# Test all cases of co-activity/activity and ensure that the weights do not explode
+# Co-activity should never be higher than the Neuron-activities multiplied
+class WeightExplosionTest(unittest.TestCase):
+    def setUp(self):
+        self.numSegments = 1
+        self.cellCount = 1
+        self.inputSize = 1
+        self.learningRate = 0.1
+        self.noise = 0.00001
+
+    # test 4 cases:
+    # no input & no bias
+    # no input
+    # no bias
+    # input & bias (+ co-activity)
```

Comment:
This test is a very good idea. Are the other tests still missing?

Reply:
I experimented around with the test, we could extend that test definitely